src/Theme_Command.php

```diff
@@ -899,4 +915,30 @@ private static function get_template_path( $template ) {
 
 		return $template_path;
 	}
+
+	/**
+	 * Check if current version of the theme is higher than the one available at WP.org
```

Comment:
WPCS - end comments with a period:
```suggestion
	 * Check if current version of the theme is higher than the one available at WP.org.
```